hstool/models.py

```diff
@@ -109,15 +109,6 @@ class DriverOfChange(GenericElement):
     time_horizon = IntegerField(choices=DOC_TIME_HORIZON_CHOICES)
     summary = TextField(null=True, blank=True)
 
-    def __unicode__(self):
-        return '{0}: {1} - {2} - {3} - {4}'.format(
-            self.name,
-            dict(DOC_TYPE_CHOICES).get(self.type, ''),
-            dict(DOC_TREND_TYPE_CHOICES).get(self.trend_type, ''),
-            dict(DOC_STEEP_CHOICES).get(self.steep_category, ''),
-            dict(DOC_TIME_HORIZON_CHOICES).get(self.time_horizon, ''),
-        )
-
```

Comment:
b/c of the listing in the assessment view. Before this change all sources and destinations were Generic, so they were using the GenericElement unicode method.